tests/Microsoft.Identity.Test.Unit.net45/CacheTests/TokenCacheTests.cs

```diff
@@ -71,7 +70,6 @@ public void GetSubsetScopesMatchedAccessTokenTest()
         {
             using (var harness = CreateTestHarness())
             {
-                harness.HttpManager.AddInstanceDiscoveryMockHandler();
```

Comment:
horary!

Reply:
hooray even!  +1 that this is fantastic.